tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/AccountTests.cs

```diff
@@ -298,6 +299,56 @@ public void TestGetAccounts()
             }
         }
 
+        [TestMethod]
+        public void TestGetAccountsMergeWithBrokerTest()
+        {
+            var tokenCacheHelper = new TokenCacheHelper();
+
+            using (var httpManager = new MockHttpManager())
+            {
+                PublicClientApplication app = PublicClientApplicationBuilder.Create(TestConstants.ClientId)
+                                                                            .WithHttpManager(httpManager)
+                                                                            .WithTelemetry(new TraceTelemetryConfig())
+                                                                            .BuildConcrete();
+                //Adding local account to local cache
+                tokenCacheHelper.PopulateCache(app.UserTokenCacheInternal.Accessor,
+                                                TestConstants.Uid,
+                                                TestConstants.Utid,
+                                                TestConstants.ClientId,
+                                                TestConstants.ProductionPrefCacheEnvironment,
+                                                "LocalAccount@microsoft.com");
```

Comment:
nit: make these constants

Reply:
Personally I like to see constants in the test, makes it more readable.

Reply:
i know you do. ;) but since they are in a few different places, makes changing them easier if that needs to happen. we tend to have more constants than not anyway. my 2 cents

Reply:
It's a good point.

Reply:
plus i knew you would comment if i commented about it. lol. ;)